.storybook/pages/AlongDemo/AlongDemo.tsx

```diff
@@ -258,7 +268,6 @@ const WatchPage = ({ onLogout }: { onLogout: () => void }) => {
             <Dropdown
               aria-label="student groups"
               buttonText={selectedOption.label}
-              // TODO: why isn't the selected styling showing up when selecting a new option?
```

Comment:
I decided I'm not going to bother investigating this because it's not really relevant to theming.